Runtime/UnityGLTFScripts.asmdef

```diff
@@ -11,7 +11,8 @@
         "Draco",
         "Unity.Meshopt.Decompress",
         "Unity.Collections",
-        "Unity.Mathematics"
+        "Unity.Mathematics",
+        "Unity.RenderPipelines.HighDefinition.Runtime"
```

Comment:
Please add a conditional check so that we dont break the plugin for someone not using HDRP.

How to do it is outlined here:
https://discussions.unity.com/t/check-if-hdrp-is-present-in-project-c/799532